CHANGELOG.md

```diff
@@ -6,6 +6,9 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [PEP 440](https://www.python.org/dev/peps/pep-0440/)
 and uses [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [8.2.1]
```

Comment:
We've made breaking changes by removing CLI options and env vars, so perhaps this should be a major release.
```suggestion
## [9.0.0]
```